### PR TITLE
Add more CSS mask- properties

### PR DESCRIPTION
**Pull request by ddbeck**
This PR adds the following CSS properties:

* [`mask-clip`](https://developer.mozilla.org/docs/Web/CSS/mask-clip)
* [`mask-composite`](https://developer.mozilla.org/docs/Web/CSS/mask-composite)
* [`mask-image`](https://developer.mozilla.org/docs/Web/CSS/mask-image)

Let me know what changes you want to see here. Thanks!

### CODE REVIEW

**Comment by ddbeck** [comment]
Oh yeah, one thing to add: `mask-composite` makes mention of `-webkit-mask-composite` which seems to be rather different and has [its own page](https://developer.mozilla.org/docs/Web/CSS/-webkit-mask-composite). Should there be a `-webkit-mask-composite.json` too?

**Comment by teoli2003** [comment]
Yes, at some point we will need to add BCD for non-standard prefix properties (like this one, but there are others). I propose we skip them for the moment as they are rather complex, not changing much and mostly deprecated.

File: css/properties/mask-image.json [hunk 1]
```diff
@@ -0,0 +1,184 @@
+{
+  "css": {
+    "properties": {
+      "mask-image": {
```
Comment (teoli2003) [inline css/properties/mask-image.json]: I think most of the comments on the current BCD should be added.
[1] Implemented in version 8.0, Chrome initially only supported the -webkit prefixed versions of gradients as values. Later, support for the unprefixed versions was added.
[2] Implemented in version 4.0, Safari initially only supported the -webkit prefixed versions of gradients as values.
[3] Android initially only supported the -webkit prefixed versions of gradients as values.
[4] iOS Safari initially only supported the -webkit prefixed versions of gradients as values

Comment (ddbeck) [reply css/properties/mask-image.json]: OK, I think I initially misinterpreted these notes as meaning that the browsers once only supported prefixed `mask-image` then gained unprefixed support. But now I don't think that I understand the notes at all. What does "prefixed versions of gradients as values" mean, exactly? Something like `-webkit-linear-gradient()`?

Comment (teoli2003) [reply css/properties/mask-image.json]: Yes, only -webkit-linear-gradient() functional was supported (and similar), not linear-gradient().

**Comment by ddbeck** [comment]
@teoli2003 OK, thanks for the explanation of the notes. I've revised the compat data and added some note text that should (hopefully) be a bit more clear and match up with the original table.

**Pull request merged**